botorch/acquisition/max_value_entropy_search.py

```diff
@@ -45,114 +53,61 @@
 CLAMP_LB = 1.0e-8
 
 
-class qMaxValueEntropy(MCAcquisitionFunction):
-    r"""The acquisition function for Max-value Entropy Search.
-
-    This acquisition function computes the mutual information of
-    max values and a candidate point X. See [Wang2018mves]_ for
-    a detailed discussion.
-
-    The model must be single-outcome.
-    q > 1 is supported through cyclic optimization and fantasies.
+class MaxValueBase(AcquisitionFunction):
```

Comment:
I've done this. But what does it do? I thought ABC would be inherited from AcquisitionFunction.

Reply:
hehe no, the fact that you're subclassing a class that's `ABC` means that it's not abstract anymore (since it's a particular subclass). So if you want that subclass again be abstract you'll want to inherit from `ABC` again.